src/app/admin/RefreshPageButton.tsx

```diff
@@ -0,0 +1,22 @@
+import Button from "@mui/material/Button";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faArrowsRotate } from "@fortawesome/free-solid-svg-icons/faArrowsRotate";
+import React from "react";
+
+const RefreshPageButton: React.FC<{}> = () => {
+    const refreshPage = (): void => {
+        window.location.reload();
```

Comment:
if at all possible I prefer NextJS specific things rather than window-specific things, in this case
```ts
const router = useRouter(); // from next/navigation
...
onClick={router.refresh}
// OR
onClick={()=>router.refresh()}
```

Reply:
I tried this previously, but I couldn't quite get it to work. The page doesn't appear to refresh when I try any of the above methods